File: 백준/Bronze/5522. 카드 게임/카드 게임.java
import java.util.*;
public class Main{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int sum = 0;
        for(int i = 1; i <= 5; i++){
            int n = sc.nextInt();
            sum += n;
        }
        System.out.println(sum);
    }
}

File: 백준/Bronze/5522. 카드 게임/카드 게임.c
#include<stdio.h>
int main(void){
    int sum = 0;
    for(int i = 1; i <= 5; i++){
        int n;
        scanf("%d",&n);
        sum += n;
    }
    printf("%d\n",sum);
    return 0;     
}